src/data/logoIcons.json

```diff
@@ -302,7 +302,7 @@
     {
       "name": "tail-9X",
       "title": "Airplane tail image for Mokulele Airlines",
-      "viewBox": "0 0 856 830"
+      "viewBox": "0 0 736 707"
```

Comment:
The Added image doesn't show anything.

Reply:
> The Added image doesn't show anything.

Concur here - when I load the SVG in this PR into https://www.svgviewer.dev/ it doesn't throw an error but the content is blank.

Reply:
The SVG added to the repo does not have attributes. This is all added in the build process via the data json file for the SVGs added to the `./dist` dir. 

A directory of icons, logos and tails can be seen here alaska.iconstails.surge.sh